tests/main/snap-auto-import-asserts/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Check that `snap auto-import` works as expected
+
+systems: [ubuntu-core-16-64]
```

Comment:
@fgimenez Can you please add those after you're back on Thursday?

@mvo5 will be out for a few days, so I'm moving this in.